ts/index.ts

```diff
@@ -27,17 +27,27 @@ export const getRegistrar = <T, C>(cfg: {
   getIdFromEvent: IdExtractionFunction,
   create: IdCreationFunction<T, C>
 }) => ({
-    update: async (events: string[]) => {
-      const keyEventId = await cfg.getIdFromEvent(events[0])
-      const previousEvents = await cfg.dbInstance.read(keyEventId) || []
+    update: async (events: string) => {
+      try {
+        const keyEventId = await cfg.getIdFromEvent(events[0])
```

Comment:
This will fail. I assume we should split the 0th event and pass it to `getIdFromEvent`, or use a different helper function.